examples/visualization/example_jupyter.ipynb

```diff
@@ -0,0 +1,166 @@
+      "text/plain": [
+       "<Figure size 1440x720 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from IPython import display\n",
+    "from loguru import logger\n",
+    "import time \n",
+    "\n",
+    "logger.remove() \n",
+    "logger.add(sys.stderr, level=\"INFO\")# using DEBUG level will create tons of text\n",
```

Comment:
Don't remove the logger! that breaks all logging, I think? Just call `paseos.set_log_level("INFO")`